src/pages/http.mdx

```diff
@@ -0,0 +1,54 @@
+export const description = 'Integrating with existing HTTP frameworks'
+
+# HTTP
+
+Nitric supports existing web app and API frameworks. The `http` resource allows you to connect nitric to your existing applications.
+
+## Enabling in your project
+
+HTTP framework support is currently in preview, you enable it in your nitric project add the following to your `nitric.yaml` file
+
+```yaml
+preview-features:
+  - http
+```
+
+## Connecting existing frameworks
+
+Here are some examples of how to connect your existing API frameworks with nitric:
+
+<CodeGroup>
+
+```javascript {{ title: "Node.js (express.js)" }}
+import { http } from '@nitric/sdk'
+import express from "express"
+
+const app = express()
+
+app.use("/hello", (req, res) => {
+  res.send("hello from express")
+});
+
+http(app)
+```
+
+```javascript {{ title: "Node.js (nestjs)", tag: "app.module.ts" }}
+import { http } from '@nitric/sdk'
+import { NestFactory } from '@nestjs/core'
+import { AppModule } from './app.module'
+
+async function bootstrap(port: number) {
+  const app = await NestFactory.create(AppModule)
+  await app.listen(port)
+}
+
+http(bootstrap)
+```
+
+```python
```

Comment:
Maybe this should be a top level <Note> instead of a comment?

Reply:
wrote the same way as custom domain support for other clouds, I think the intention was to convey we're planning support soonish